CONTRIBUTING.md

```
@@ -0,0 +1,125 @@
+
+## Installing the correct versions of Java, Maven and necessary dependencies
+
+Please make sure your development environment complies with the requirements discussed under the relevant section of the parent project's [CONTRIBUTING](https://github.com/BlackBeltTechnology/judo-community/blob/develop/CONTRIBUTING.adoc) guide.
+
+## Code Structure
+
+This project follows a standard Java project structure, governed by Maven, with potential Maven submodules.
+
+### Project Layout
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)